homeassistant/components/climate/fibaro.py

```diff
@@ -0,0 +1,272 @@
+"""
+Support for Fibaro thermostats.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.fibaro/
+"""
+import logging
+
+from homeassistant.components.climate import (
+    ClimateDevice, STATE_AUTO, STATE_COOL,
+    STATE_DRY, STATE_FAN_ONLY, STATE_HEAT,
+    ENTITY_ID_FORMAT,
+    SUPPORT_TARGET_TEMPERATURE,
+    SUPPORT_OPERATION_MODE, SUPPORT_FAN_MODE)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    STATE_OFF,
+    TEMP_CELSIUS,
+    TEMP_FAHRENHEIT)
+
+from homeassistant.components.fibaro import (
+    FIBARO_DEVICES, FibaroDevice)
+
+SPEED_LOW = 'low'
+SPEED_MEDIUM = 'medium'
+SPEED_HIGH = 'high'
+STATE_AUXILIARY = 'auxiliary'
```

Comment:
But these devices are managed by a Fibaro hub, so if they are in a state that is not support by HA, I should still understand that, I think. The opmode states are coming from the zwave specifications. So I'm not sure what to do in case HA does not support an opmode state that the device is in.

Reply:
We can only map as best as possible to the operation modes that home assistant supports. If those don't make sense we have to leave the other modes out.

It's ok to add device state attributes that show extra information related to the state.

Reply:
Is there a list somewhere of supported modes by HA?